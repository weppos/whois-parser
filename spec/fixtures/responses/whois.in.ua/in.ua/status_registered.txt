% This is the SUNIC WHOIS server.
% The Whois is subject to Terms of use
% See https://hostmaster.ua/documents.php
%
% In the process of delegation of a domain name,
% the Registrant is an entity who uses and manages a certain domain name,
% and the Registrar is a business entity that provides the Registrant
% with the services necessary for the technical maintenance of the
% registration and operation of the domain name.
% For information on the Registrant's domain name, you should contact the Registrar.

domain:             dle.in.ua
dom-public:         NO
mnt-by:             ua.nic
nserver:            ns10.uadns.com
nserver:            ns12.uadns.com
nserver:            ns11.uadns.com
status:             ok
created:            2007-12-18 20:12:35+02
modified:           2019-12-03 11:15:10+02
expires:            2020-12-20 00:00:00+02
source:             SUNIC

% Registrar:
% ==========
registrar:          ua.nic
organization:       NIC.UA LLC
organization-loc:   ТОВ «НІК.ЮЕЙ»
url:                http://nic.ua
city:               Dnipro
country:            UA
abuse-email:        abuse@nic.ua
abuse-phone:        +380.445933222
abuse-postal:       Ukraine 49000 Dnipro PO/BOX 80
abuse-postal-loc:   UA 49000 Днепр а/я 80
source:             SUNIC

% Registrant:
% ===========
person:             n/a
organization:       n/a
address:            n/a
person-loc:         not published
organization-loc:   not published
address-loc:        not published
email:              not published
phone:              not published
fax:                not published
mnt-by:             sunic
status:             linked
status:             ok
created:            2017-09-30 01:31:30+03
modified:           2017-09-30 01:31:30+03
source:             SUNIC

% Administrative Contacts:
% ========================
person:             n/a
organization:       n/a
address:            n/a
person-loc:         not published
organization-loc:   not published
address-loc:        not published
email:              not published
phone:              not published
fax:                not published
mnt-by:             sunic
status:             linked
status:             ok
created:            2017-09-30 01:31:30+03
modified:           2017-09-30 01:31:30+03
source:             SUNIC


% Technical Contacts:
% ===================
person:             NIC-UANIC
organization:       NIC.UA LLC
address:            not published
person-loc:         n/a
organization-loc:   n/a
address-loc:        n/a
email:              support@nic.ua
phone:              +380.445933222
fax:                +380.445937569
mnt-by:             sunic
status:             linked
status:             ok
created:            2016-09-18 10:37:53+03
modified:           2016-09-17 10:37:53+03
source:             SUNIC

