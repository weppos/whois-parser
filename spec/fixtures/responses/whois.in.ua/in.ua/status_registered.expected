#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2007-12-18 20:12:35 +02:00}

#updated_on
  %s %CLASS{time}
  %s %TIME{2019-12-03 11:15:10 +02:00}

#expires_on
  %s %CLASS{time}
  %s %TIME{2020-12-20 00:00:00 +02:00}


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns10.uadns.com"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns12.uadns.com"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns11.uadns.com"
